pom.xml

```diff
@@ -202,7 +202,7 @@
     <test.include.tags></test.include.tags>
 
     <!-- Package to use when relocating shaded classes. -->
-    <spark.shade.packageName>org.spark_project</spark.shade.packageName>
+    <spark.shade.packageName>org.spark.project</spark.shade.packageName>
```

Comment:
This one doesn't need to change.

Reply:
can you elaborate @vanzin  - `common/network-yarn` package uses this property from parent pom to get the shade prefix

Reply:
Hmm, I see what the other one is doing now. My bad.